v5/Models/PostValues/PostDatetime.php

```diff
@@ -2,23 +2,40 @@
 
 namespace v5\Models\PostValues;
 
-use Illuminate\Database\Eloquent\Model;
-use Illuminate\Support\Facades\Validator;
 use v5\Models\Helpers\HideTime;
 
 class PostDatetime extends PostValue
 {
     public $table = 'post_datetime';
 
+    /**
+     * @inheritdoc
+    */
+    protected $fillable = [
+        'post_id',
+        'form_attribute_id',
+        'value',
+        'metadata',
+    ];
+
+     /**
+     * The attributes that should be cast.
+     *
+     * @var array
+     */
+    protected $casts = [
+        'metadata' => 'array',
+        'value' => 'date',
```

Comment:
Isn't this line overridden by the accessor `getValueAttribute` and mutator `setValueAttribute` below?

Reply:
Yeah you're right ... it overrides the cast